src/library/Server/Manager/Hestia.php

```diff
@@ -42,7 +38,27 @@ public function _getPort()
     public static function getForm()
     {
         return [
-            'label' => 'Hestia',
+            'label' => 'Hestia Control Panel',
+            'form' => [
+                'credentials' => [
+                    'fields' => [
+                        [
+                            'name' => 'username',
```

Comment:
For Hestia it should be

Access key and secret key

Reply:
What is a secret key?

Is it username + (access key/secret key)?

Reply:
Until we implement custom fields to the backend, I’ll pass them as “username” and “accesshash”.

The end user will see them as “Access key ID” and “Secret key”, but the backend will treat them as the username and the access hash. If that’s okay, I’ll do it.

Reply:
That is fine.. If we change we need also update:

https://github.com/FOSSBilling/FOSSBilling/blob/7bfdf2a42c3ea7d14dcfee4d769534137d78fe12/src/library/Server/Manager/Hestia.php#L89-L95

Other wise it will not able to connect any more it will be come:

```
$params['hash'] = $this->_config['username'].':'.$this->_config['accesshash'];
```

Assuming we want all users to change it. 

Ideally we should add a link to the documentation about at the Hestia docs in this case.

Reply:
@jaapmarcus Would this be fine?

![image](https://user-images.githubusercontent.com/35808275/206275031-c5416011-37bb-4a51-8884-69424b69b76c.png)

I'm planning to extend the form more and add stuff like documentation links in a future PR. It definitely is a great idea.

Also, should I do the same for Vesta or are there any differences on how they work?

Reply:
Yes they only support: username / password (admin / password)

Reply:
Alright. Changed it for Hestia and left Vesta with username/password based authorization.